webpack/css.js

```diff
@@ -0,0 +1,2 @@
+// Empty (needed for webpack)
+import '../components/style.css';
```

Comment:
This isn't empty. Do we need to remove that comment? Or is the css no longer needed here?

Reply:
Yea, we should remove this